lib/pry/input_completer.rb

```diff
@@ -239,4 +227,39 @@ def build_path(input)
     end
     return path, input
   end
+
+  private
+
+  def self.fetch_obj_space_collection
```

Comment:
@printercu There is no problem to refactor it, but as I see, this code was written more than 3 month ago. Now, unfortunately, I see, that this chunk is still unuseful for repo owner. If it helps somebody- awesome!

You may take this snippet and make own PW with refactored code, if you wish.

About singleton in the bottom, you, probably, missed, that it is in private section. I prefer put all private methods in 1 section.